tests/Unit/Parallel/Test_PhaseChangeMain.cpp

```diff
@@ -20,6 +20,7 @@
 #include "Parallel/Main.hpp"
 #include "Parallel/ParallelComponentHelpers.hpp"
 #include "Parallel/PhaseControl/PhaseControlTags.hpp"
+#include "Parallel/PhaseControlReductionHelpers.hpp"
```

Comment:
The two algorithm tests `Test_AlgorithmPhaseControl.cpp` and `Test_PhaseChangeMain.cpp` probably need to link `PhaseControl`.